PerimeterXModule/Internals/Helpers/RemoteServer.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Configuration;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace PerimeterX
+{
+	internal class RemoteServer
+	{
+		string _remoteUrl;
+		HttpContext _context;
```

Comment:
Why use the `_` convention? I think we can stick with our current approach..

Reply:
👍